### PR TITLE
Add orphan check for server, volume and fip

### PR DESCRIPTION
**Pull request by wangxiyuan**
Add a new plugin into labchecker for orphan check.

### CODE REVIEW

File: openlabcmd/openlabcmd/plugins/jobs/image.py [hunk 1]
```diff
@@ -4,10 +4,12 @@
...
 class ImagePlugin(Plugin):
-
     ptype = 'jobs'
     name = 'image'
 
+    def __init__(self, cloud, config):
+        super(ImagePlugin, self).__init__(cloud, config)
+
     def check(self):
         self.failed = False
```
Comment (Yikun) [inline openlabcmd/openlabcmd/plugins/jobs/image.py:14]: we already set the `failed` and `reasons` in parent class, we can remove these now.

File: openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+
+class OrphanResourcePlugin(Plugin):
+    ptype = 'nodepool'
+    name = 'orphan_resource'
+
+    # This Plugin is only for checking, as it's a dangerous action for each.
+    def __init__(self, cloud, config):
+        super(OrphanResourcePlugin, self).__init__(cloud, config)
+        self.openstack_client = openstack.connect(cloud)
```
Comment (Yikun) [inline openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py:17]: In future we can move the openstack_client into a separate class, and the plugin is using the openstacksdk will be defined as:
```
class OrphanResourcePlugin(Plugin, OpenstackSDKHelper):
```

**Review by Yikun — APPROVED** [review]

File: openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py (hunk 2 above)
Comment (Yikun) [reply openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py:17]: I mean if we have lots of other methods use sdk to do some complex thing, but now we can keep it.

File: openlabcmd/openlabcmd/plugins/jobs/image.py (hunk 1 above)
Comment (wangxiyuan) [reply openlabcmd/openlabcmd/plugins/jobs/image.py:14]: It's more like a `tag reset button`.

File: openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py (hunk 2 above)
Comment (wangxiyuan) [reply openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py:17]: Good point, it can be done once more plugin using openstacksdk

File: openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        self.vm_white_list = list(set(vm_common_white_list +
+                                      vm_addition_white_list))
+        self.volume_white_list = self.config.get(
+            'check', 'volume_white_list',
+            fallback='').replace(' ', '').split(',')
+
+        self.fip_white_list = self.config.get(
+            'check', 'fip_white_list', fallback='').replace(' ', '').split(',')
+
+    def _is_check_heart_beat_overtime(self, timestamp):
```
Comment (bzhaoopenstack) [inline openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py]: This function is meaningless. How about "_is_overtime" or something like this? This is not related with heartbeat.

File: openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        if not timestamp:
+            return True
+        timeout = RESOURCE_TIMEOUT
```
Comment (bzhaoopenstack) [inline openlabcmd/openlabcmd/plugins/nodepool/orphanresource.py]: I saw the whitelist is moved to cfg. How about to do the same with timeout?

**Review by bzhaoopenstack — APPROVED** [review]
lgtm
